packages/react/ds-core-form/src/ui/Field/common/Wrapper/withWrapper.ts

```diff
@@ -1,16 +1,18 @@
 import * as React from "react";
 import { useMemo } from "react";
-import type { TextProps } from "../../inputs/index.js";
 import type {
   BaseInputProps,
   BaseWrapperProps,
   WrappedComponentProps,
   WrapperProps,
 } from "../../types.js";
 import DefaultWrapper from "./Wrapper.js";
+import withConditionalDisplay from "./withConditionalDisplay.js";
 
 type WrappedComponentPropsInternal<ComponentProps extends BaseInputProps> =
-  Omit<WrappedComponentProps<ComponentProps>, "Component">;
+  Omit<WrappedComponentProps<ComponentProps>, "Component"> & {
+    condition?: [string[], (depsValues: unknown[]) => boolean];
```

Comment:
can you extract this condition type into its own named type, so that this is more readable?